Tools/autotest/arduplane.py

```diff
@@ -3046,6 +3046,28 @@ def WatchdogHome(self):
         if ex is not None:
             raise ex
 
+    def fly_landing_baro_drift(self):
+
+        self.customise_SITL_commandline([], wipe=True)
+        self.set_parameter("SIM_BARO_DRIFT", -0.02)
+
+        self.set_analog_rangefinder_parameters()
+
+        self.reboot_sitl()
+
+        # Enable rangefinder for landing (Plane only!)
+        self.set_parameter("SIM_TERRAIN", 0)
+
+        self.set_parameter("RNGFND_LANDING", 1)
+        self.set_parameter("RNGFND1_MAX_CM", 4000)
+        self.set_parameter("LAND_SLOPE_RCALC", 2)
+        self.set_parameter("LAND_ABORT_DEG", 2)
```

Comment:
```suggestion
        self.set_parameters({
            "SIM_TERRAIN": 0,
            "RNGFND_LANDING": 1,
            "RNGFND1_MAX_CM": 4000,
            "LAND_SLOPE_RCALC": 2,
            "LAND_ABORT_DEG": 2,
        })
```

It might be worth moving this to before the reboot, and combine with the other param-set - leaves a cleaner state for the mission flight.

This isn't a blocker for this to go in in any way - just a tidy-up

Reply:
Thanks, adopted.